CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to GlueStick
+
+GlueStick is one of TrueCar's open source projects that is build to speed up the development process internally in [truecar.com](https://truecar.com) and also help a larger audience reduce the time needed to build apps. It allows the developer to create and deploy enterprise-grade React and Redux application right from your command line.
+
+### Pull Requests
+
+TrueCar team will be monitoring for pull requests.
+
+**You can submit your pull request on the `develop` branch**.
+
+Before submitting a pull request, please make sure the following is done…
+
+1. Fork the repo and create your branch from `develop`.
+2. **Describe your test plan in your commit.** If you've added code that should be tested, add tests!
+3. If you've changed APIs, update the documentation.
+4. Ensure tests, lints and flow-check pass on *Circle CI*.
+5. Ensure that *Node Security* check passed.
+6. Squash your commits (`git rebase -i`).
+   one intent alongs with one commit makes it clearer for people to review and easier to understand your intention.
+
+### Contributor License Agreement (CLA)
+
+TODO: @toddw
```

Comment:
Talking to people about setting up a CLA bot like https://github.com/clabot/clabot. Anyone have other suggestions or experience with cla bot?